examples/hello_world/my_script.py

```diff
@@ -7,37 +6,36 @@
 from hamilton import driver
 
 logging.basicConfig(stream=sys.stdout)
-initial_columns = {  # load from actuals or wherever -- this is our initial data we use as input.
-    # Note: these values don't have to be all series, they could be a scalar.
-    "signups": pd.Series([1, 10, 50, 100, 200, 400]),
-    "spend": pd.Series([10, 10, 20, 40, 40, 50]),
-}
+
 # we need to tell hamilton where to load function definitions from
-# programmatic code to load modules:
-# module_name = "my_functions"
-# my_functions = importlib.import_module(module_name)
-# or import module(s) directly:
+# we use module pointers to do this
 import my_functions
 
-dr = driver.Driver(initial_columns, my_functions)  # can pass in multiple modules
-# we need to specify what we want in the final dataframe. These can be string names, or function references.
+# This uses the module above, but you can pass in as many as you want
+dr = driver.Driver({}, my_functions)
```

Comment:
we should really change this to the builder pattern.

Reply:
Done, but it makes it slightly more complicated